app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -226,6 +228,29 @@ def security_groups
     end
   end
 
+  def cloud_databases
+    collector.resources.each do |resource|
+      persister.cloud_databases.build(
+        :ems_ref => resource["guid"],
+        :name    => resource["name"],
+        :status  => resource["state"]
```

Comment:
pending ticket approval, we can add stuff like `db_engine`, `used_storage`, and `max_storage`